src/include/common/container/bitmap.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
+#include <memory>
+#include "common/common_defs.h"
+#include "common/concurrent_bitmap.h"
+
+#ifndef BYTE_SIZE
+#define BYTE_SIZE 8u
+#endif
+
+// Some platforms would have already defined the macro. But its presence is
+// not standard and thus not portable. Pretty sure this is always 8 bits.
+// If not, consider getting a new machine, and preferably not from another
+// dimension :)
+static_assert(BYTE_SIZE == 8u, "BYTE_SIZE should be set to 8!");
```

Comment:
Seeing how this is duplicated, maybe the right way to do this is to have ConcurrentBitmap inherit from Bitmap, (but then it becomes unclear whether a method should be thread-safe or not.).

At the very least, it makes sense that concurrent_bitmap.h can include the bitmap header to get the same #defines.